src/sql/migrations.rs

```diff
@@ -468,6 +468,22 @@ paramsv![]
         recode_avatar = true;
         sql.set_db_version(77).await?;
     }
+    if dbversion < 78 {
+        // move requests but the last one to "Archived Chats";
+        // this way, the app looks familiar after the contact request upgrade.
+        info!(context, "[migration] v78");
+        sql.execute_migration(
+            "UPDATE chats
+                SET archived=1
+              WHERE blocked=2
+                AND id!=COALESCE((SELECT chat_id FROM msgs m
```

Comment:
Maybe remove this `AND` and simply archive all the contact requests?

Reply:
yip, make sense, esp. after recent discussions on irc (maybe wont ux expectations, so there is very few or maybe even now advantage from that complicated code). thanks a lot for your points here, @link2xt!